### PR TITLE
Fishy packages

### PR DESCRIPTION
**Pull request by bpinto**

### CODE REVIEW

File: pkg/omf/cli/omf.remove_package.fish [hunk 1]
```diff
@@ -15,7 +15,7 @@ function omf.remove_package
   for path in {$OMF_PATH,$OMF_CONFIG}/{pkg}/$pkg
     not test -d $path; and continue
 
-    emit uninstall_$pkg
+    source $path/uninstall.fish; and emit uninstall_$pkg
```
Comment (bobthecow) [inline pkg/omf/cli/omf.remove_package.fish]: for backwards compatibility for now, we should emit the event even if we can't source the uninstall file. drop it down to the next line with a TODO to change it to this way after a while?

**Comment by bobthecow** [comment]
:+1: except the backwards compatibility break.

File: pkg/omf/cli/omf.remove_package.fish (hunk 1 above)
Comment (bpinto) [inline pkg/omf/cli/omf.remove_package.fish]: Yeah... but we have no plugins that have a `uninstall` function?

Comment (bobthecow) [inline pkg/omf/cli/omf.remove_package.fish]: oh, i don't know. i haven't checked.

**Pull request merged**